app/Models/Period.php

```diff
@@ -37,4 +37,17 @@ public function scopeCurrent($query)
             ->where('end_date', '>=', $currentDate->toDateString())
             ->where('status', PostingPeriodStatusType::OPEN->value);
     }
+
+    public function scopeCheckIfStatusIsOpenMonthly($query)
```

Comment:
scope should not include updates
this business logic should be included in service